docs/how-to-guides/config-settings.rst

```diff
@@ -114,7 +114,7 @@ shows the detailed Meson setup log when the build fails:
     - name: Build the package
       run: python -m build --wheel -Cbuild-dir=build
     - name: Show meson-log.txt
-      if: failure()
+      if: ${{ failure() }}
```

Comment:
I think both versions work, but GHA docs now consistently use the `${{ }}` syntax so this is a good change.